db/payment_icons.yml

```diff
@@ -994,3 +994,27 @@
   name: nelo
   label: Nelo
   group: other
+-
+  name: vvv_giftcard
+  label: VVV Cadeaukaart
+  group: other
+  -
```

Comment:
We need to move these `-` back to align with all other previous `-`.

Reply:
Should be fixed now.. :)